yaptide/batch/fabric_test.py

```diff
@@ -1,8 +1,18 @@
 from fabric import Connection, Result
+from paramiko import Ed25519Key
 
-login = "---------------"
-host = "ares"
-connection = Connection(host=f'{login}@{host}.cyfronet.pl', connect_kwargs={"password": "---------------"})
+import os
+
+from pathlib import Path
+
+ROOT_DIR = os.path.dirname(os.path.realpath(__file__))
+ssh_key_path = Path(ROOT_DIR, "id_ed25519")
```

Comment:
with pathlib this would become:
```suggestion
ssh_key_path = ROOT_DIR / "id_ed25519"
```